crates/hc_bundle/schema/dna-manifest.schema.json

```diff
@@ -68,14 +68,6 @@
           }
         }
       }
-    },
-    "lineage": {
```

Comment:
Removal of this means that an `hc` built with `unstable-dpki` will display a wrong dna manifest schema in the `hc dna schema` command (i.e. one without the lineage field). To prevent this we'd need to add a dedicated unstable-dpki json schema file and point to that instead in the unstable-dpki case in hc_bundle. I consider this not worth it in that case.

Reply:
> To prevent this we'd need to add a dedicated unstable-migration json schema file and point to that instead in the unstable-migration case in hc_bundle. I consider this not worth it in this case.

Seems straightforward -- I think we might as well.

Reply:
It's not fully as straight forward because of [this](https://github.com/holochain/holochain/blob/4b4fbe228d3779adca7defcec49fe2532ebdfddb/crates/hc_bundle/tests/test_cli.rs#L410-L412) function which is agnostic to the schema kind (dna, happ, web-happ) and so it would need some fiddling. Maybe that's worth it...will think about some options.

Reply:
addressed 86b5672b14e92d8a243968eda058297694c46a21